src/i18n/en_AU/buildings.ts

```diff
@@ -188,7 +188,7 @@ export default {
         ],
         levelcost: ['1.-20. 19.000 Credits / 11 Coins'],
         maxBuildings: 'No limit',
-        maxLevel: 20,
+        maxLevel: 30,
```

Comment:
that is false 
![image](https://user-images.githubusercontent.com/1109899/184522047-fbd4fe24-36b0-41b3-a373-6061307bf83b.png)